docs/pipeline-components-and-applications/snowplow-mini/usage-guide/index.md

```diff
@@ -16,15 +16,9 @@ Jump to [First Time Usage](#first-time-usage) if it is your first time with a Mi
 
 ## Upgrading
 
-Mini 0.8.0 updated some of the internal components resulting in some breaking changes as following:
+Mini 0.15.0 replaced Elasticsearch with Opensearch. The only breaking change this migration would bring is the removal of mapping types. It means that you don't have to provide a mapping type in your search queries anymore when accessing your data in good or bad indices. For example, good events count can be found using the following endpoint in previous versions: `/elasticsearch/good/good/_count`. Starting with 0.15.0, it can be found using this endpoint: `/elasticsearch/good/_count`.
```

Comment:
Copy some sentences with the reasoning from the release announcement, to cover these points:
* Opensearch is an Apache 2 fork of Elastic
* We switched to it so that we can keep getting the security patches